fbw-common/src/wasm/fbw_common/src/ThrottleAxisMapping.cpp

```diff
@@ -326,14 +346,20 @@ void ThrottleAxisMapping::storeConfigurationInLocalVariables(const Configuration
   idDetentClimbHigh->set(configuration.climbHigh);
   idDetentFlexMctLow->set(configuration.flxMctLow);
   idDetentFlexMctHigh->set(configuration.flxMctHigh);
-  idDetentTogaLow->set(configuration.togaLow);
-  idDetentTogaHigh->set(configuration.togaHigh);
+  if (configuration.useTogaOnAxis) {
+    idDetentTogaLow->set(configuration.togaLow);
+    idDetentTogaHigh->set(configuration.togaHigh);
+  } else {
+    idDetentTogaLow->set(0.0);
+    idDetentTogaHigh->set(0.0);
+  }
 }
 
 ThrottleAxisMapping::Configuration ThrottleAxisMapping::loadConfigurationFromIniStructure(const INIStructure& structure) {
   idUsingConfig->set(true);
   return {
       INITypeConversion::getBoolean(structure, CONFIGURATION_SECTION_COMMON, "REVERSE_ON_AXIS", false),
+      INITypeConversion::getBoolean(structure, CONFIGURATION_SECTION_COMMON, "TOGA_ON_AXIS", false),
```

Comment:
TOGA on axis should be the default.

Reply:
Should reverse on axis be true on default as well? As it is on line 300

Reply:
Nothing should change for existing configs. Making this new option default (toga on axis off) would break tens of thousands of setups.